<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>331</width>
        <height>621</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>331</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="updateBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>470</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kpSpinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ki</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kiSpinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.040000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kdSpinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>240</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kd</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clearBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>550</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Target</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="targetSpinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>330</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>99.989999999999995</double>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="minimumSize">
       <size>
        <width>721</width>
        <height>611</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>627</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1282</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
